packages/actions/resources/lang/en/import.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+return [
+
+    'label' => 'Import :label',
+
+    'modal' => [
+
+        'heading' => 'Import :label',
+
+        'form' => [
+
+            'file' => [
+                'label' => 'File',
+                'placeholder' => 'Upload a .csv file',
+            ],
+
+            'columns' => [
+                'label' => 'Columns',
+                'placeholder' => 'Select a column',
+            ],
+
+        ],
+
+        'actions' => [
+
+            'download_example' => [
+                'label' => 'Download an example .csv file.',
+            ],
+
+            'import' => [
+                'label' => 'Start import',
+            ],
+
+        ],
+
+    ],
+
+    'notifications' => [
+
+        'completed' => [
+
+            'title' => 'Import completed',
+
+            'actions' => [
+
+                'download_failed_rows_csv' => [
+                    'label' => 'Download information about the failed row|Download information about the failed rows',
```

Comment:
Haven't checked the UI yet but these labels are quite long?

Reply:
Its ok, the button has its own line

Reply:
Alright, maybe just remove `the` from the label then?

Reply:
Hm I think this reads better

Reply:
But length..? 😅
Could you post a screenshot of what the UI looks like?

Reply:
It fits on one line in the database notifications slideover

Reply:
Alright.